README.md

```diff
@@ -30,37 +30,9 @@ environment with Python 3.6, you can install one via [Miniconda3](https://conda.
   pushd translate
   ```
 
-- Build [PyTorch](https://pytorch.org/) from source (currently needed for ONNX compatibility):
+- Install [PyTorch nightly build](https://pytorch.org/) (currently needed for ONNX compatibility):
   ```bash
-  # Set to 9 if you have CUDA 9.
-  TMP_CUDA_VERSION="8"
-
-  # Uninstall previous versions of PyTorch. Doing this twice is intentional.
-  # Error messages about torch not being installed are benign.
-  pip uninstall -y torch
-  pip uninstall -y torch
-
-  # Install basic PyTorch dependencies.
-  conda install -y cffi cmake mkl mkl-include numpy pyyaml setuptools typing
-  # Add LAPACK support for the GPU.
-  conda install -y -c pytorch "magma-cuda${TMP_CUDA_VERSION}0"
-
-  # Install NCCL2.
-  wget "https://s3.amazonaws.com/pytorch/nccl_2.1.15-1%2Bcuda${TMP_CUDA_VERSION}.0_x86_64.txz"
-  TMP_NCCL_VERSION="nccl_2.1.15-1+cuda${TMP_CUDA_VERSION}.0_x86_64"
-  tar -xvf "${TMP_NCCL_VERSION}.txz"
-  export NCCL_ROOT_DIR="$(pwd)/${TMP_NCCL_VERSION}"
-  export LD_LIBRARY_PATH="${NCCL_ROOT_DIR}/lib:${LD_LIBRARY_PATH}"
-  rm "${TMP_NCCL_VERSION}.txz"
-
-  # Build PyTorch from source.
-  git clone --recursive https://github.com/pytorch/pytorch
-  pushd pytorch
-  # Pin to a specific commit for now to guard against breakage in HEAD.
-  git checkout d154d32
-  git submodule update --init
-  NCCL_ROOT_DIR="${NCCL_ROOT_DIR}" python3 setup.py install \
-    2>&1 | tee PYTORCH_OUT
+  conda install pytorch-nightly -c pytorch
   ```
 
 - Build [Caffe2](http://caffe2.ai/) from source (under PyTorch):
```

Comment:
I think the main reason that we reverted to building PyTorch from source was b/c @jhcross found some incompatibility between the nightly builds of Caffe2/PyTorch and ONNX? Has that been resolved? (@jhcross - can you elaborate what the issue was again?)